x-pack/plugins/apm/public/components/app/transaction_overview/index.tsx

```diff
@@ -73,50 +58,10 @@ export function TransactionOverview() {
       <TransactionCharts />
       <EuiSpacer size="s" />
       <EuiPanel hasBorder={true}>
-        <EuiTitle size="xs">
-          <h3>Transactions</h3>
-        </EuiTitle>
-        <EuiSpacer size="s" />
-        {!transactionListData.isAggregationAccurate && (
-          <EuiCallOut
-            title={i18n.translate(
-              'xpack.apm.transactionCardinalityWarning.title',
-              {
-                defaultMessage:
-                  'This view shows a subset of reported transactions.',
-              }
-            )}
-            color="danger"
-            iconType="alert"
-          >
-            <p>
-              <FormattedMessage
-                id="xpack.apm.transactionCardinalityWarning.body"
-                defaultMessage="The number of unique transaction names exceeds the configured value of {bucketSize}. Try reconfiguring your agents to group similar transactions or increase the value of {codeBlock}"
-                values={{
-                  bucketSize: transactionListData.bucketSize,
-                  codeBlock: (
-                    <EuiCode>xpack.apm.ui.transactionGroupBucketSize</EuiCode>
-                  ),
-                }}
-              />
-
-              <ElasticDocsLink
-                section="/kibana"
-                path="/troubleshooting.html#troubleshooting-too-many-transactions"
-              >
-                {i18n.translate(
-                  'xpack.apm.transactionCardinalityWarning.docsLink',
-                  { defaultMessage: 'Learn more in the docs' }
-                )}
-              </ElasticDocsLink>
-            </p>
-          </EuiCallOut>
-        )}
-        <EuiSpacer size="s" />
-        <TransactionList
-          isLoading={transactionListStatus === 'loading'}
-          items={transactionListData.items || []}
+        <ServiceOverviewTransactionsTable
```

Comment:
Why is it still called `ServiceOverviewTransactionsTable` if it's not on the service overview?

Reply:
Thanks, I actually forgot to move it to the `shared` folder and rename it.

Reply:
Fixed it here: https://github.com/elastic/kibana/pull/107299/commits/b8209e6b5024c90f28e491f87bf26b9985ebb810